charts/portal/values.yaml

```diff
@@ -35,6 +35,9 @@ centralidp:
     miw: "Cl5-CX-Custodian"
     technicalRolesManagement: "technical_roles_management"
 
+# -- Provide ssi credential base address.
+ssiCredentialAddress: "https://ssi-credential-issuer.example.org"
```

Comment:
you don't need to introduce a new value, it's already there in the values.yaml file as `issuerComponentAddress` because the backend already needed the address. Please change to that value.